code/mob/living/critter/gunbot.dm

```diff
@@ -93,3 +94,46 @@
 
 	get_disorient_protection()
 		return max(..(), 80)
+
+	attack_hand(mob/user)
+		user.lastattacked = src
+		if(!user.stat)
+			if (user.a_intent != INTENT_HELP)
+				actions.interrupt(src, INTERRUPT_ATTACKED)
+			switch(user.a_intent)
+				if(INTENT_HELP) //Friend person
+					playsound(src.loc, 'sound/impact_sounds/Generic_Shove_1.ogg', 50, 1, -2)
+					user.visible_message("<span class='notice'>[user] gives [src] a [pick_string("descriptors.txt", "borg_pat")] pat on the [pick("back", "head", "shoulder")].</span>")
+				if(INTENT_DISARM) //Shove
+					SPAWN_DBG(0) playsound(src.loc, 'sound/impact_sounds/Generic_Swing_1.ogg', 40, 1)
+					user.visible_message("<span class='alert'><B>[user] shoves [src]! [prob(40) ? pick_string("descriptors.txt", "jerks") : null]</B></span>")
+				if(INTENT_GRAB) //Shake
+					if (istype(user, /mob/living/carbon/human/machoman))
+						. = ..()
+						return
+					playsound(src.loc, 'sound/impact_sounds/Generic_Shove_1.ogg', 30, 1, -2)
+					user.visible_message("<span class='alert'>[user] shakes [src] [pick_string("descriptors.txt", "borg_shake")]!</span>")
+				if(INTENT_HARM) //Dumbo
+					if (istype(user, /mob/living/carbon/human/machoman))
+						. = ..()
+						return
+					if (ishuman(user) && !ishunter(user) && !iswerewolf(user) && !iswrestler(user))
+						if (user.is_hulk())
+							src.TakeDamage("All", 5, 0)
+							if (prob(20))
+								var/turf/T = get_edge_target_turf(user, user.dir)
+								if (isturf(T))
+									src.visible_message("<span class='alert'><B>[user] savagely punches [src], sending them flying!</B></span>")
+									src.throw_at(T, 10, 2)
+								else
+									src.visible_message("<span class='alert'><B>[user] punches [src]!</B></span>")
+							playsound(src.loc, pick(sounds_punch), 50, 1, -1)
+						/*if (user.glove_weaponcheck())
+							user.energyclaws_attack(src)*/
```

Comment:
```suggestion
```
?

Reply:
Again copied from robots. Presumably can be removed.
https://github.com/goonstation/goonstation/blob/02c202ec1f52df4dc72e723dc8f989a8828e4997/code/mob/living/silicon/robot.dm#L1536

Reply:
yea pls remove thanks